albumentations/augmentations/geometric/transforms.py

```diff
@@ -1100,20 +1134,25 @@ def apply_to_mask(
             value=self.mask_value,
         )
 
-    def apply_to_bbox(
+    def apply_to_bboxes(
         self,
-        bbox: BoxInternalType,
+        bboxes: Sequence[BoxType],
```

Comment:
If we start to use ndarray for bboxes and keypoints we do not need to transform them to arrays once inside `Compose`